### PR TITLE
Feat/bac 52 auth restructuring

### PR DESCRIPTION
**Pull request by ibkay998**
[Linear Ticket BAC-52 BAC-56](https://linear.app/team-bevel/issue/BAC-52/inject-auth-dependency-into-router)

Description
- Created the Auth route in charge of authenticating the admin.
- Added endpoint to create admin user.
- Added unit test

sample request ```POST /api/auth  body=email: str password: str```
sample response 
```
{
  "access_token": str,
  "token_type": "bearer"
}
```

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,4 @@
 API_V1_STR="/api"
 PROJECT_NAME="FastAPI"
```
Comment (ibkay998) [inline .env.example:2]: delete the .env.example

File: backend/alembic/env.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+from logging.config import fileConfig
```
Comment (ibkay998) [inline backend/alembic/env.py:1]: delete file

File: backend/alembic/script.py.mako [hunk 3]
```diff
@@ -0,0 +1,24 @@
+"""${message}
```
Comment (ibkay998) [inline backend/alembic/script.py.mako:1]: delete file

File: backend/app/api/v1/admin.py [hunk 4]
```diff
@@ -8,24 +8,23 @@
...
+    Create Admin
+    """
+    admin = crud.admin.get_by_email(db, email=admin_in.email)
+    if admin:
+        raise HTTPException(
+            status_code=400,
+            detail="Admin with email already exists."
+        )
+    admin = crud.admin.create(db, obj_in=admin_in)
+    return admin
```
Comment (ibkay998) [inline backend/app/api/v1/admin.py:27]: add success to the return

File: backend/testing.sql [hunk 5]
```diff
@@ -1 +1,2 @@
-SELECT * FROM currencies
\ No newline at end of file
+INSERT INTO admins (email, password, is_active)
+VALUES ('ibukun46@gmail.com', '$2b$12$EixZaYVK1fsbw1ZfbX3OXePaWxn96p36WQoeG6Lruj3vjPGga31lW',True);
```
Comment (ibkay998) [inline backend/testing.sql:2]: remove content

**Review by alisani081 — APPROVED** [review]

**Review by jpphilips — APPROVED** [review]

**Pull request merged**